visualstudio/2017/netstandard/NetstandardModuleSample/NetstandardModuleSample/module_dev_sample.json

```diff
@@ -0,0 +1,55 @@
+﻿{
+    "loaders": [
+        {
+            "type": "dotnetcore",
+            "name": "dotnetcore",
+            "configuration": {
+                "binding.path": "dotnetcore.dll",
+                "binding.coreclrpath": "C:\\Program Files\\dotnet\\shared\\Microsoft.NETCore.App\\1.1.1\\coreclr.dll",
```

Comment:
This sample is for VS2017... this file is also from the IoT Edge SDK repo.

Visual Studio 2017 will install and run on the following operating systems:

Windows 10 version 1507 or higher: Home, Professional, Education, and Enterprise (LTSB is not supported)
Windows Server 2016: Standard and Datacenter
Windows 8.1 (with Update 2919355): Basic, Professional, and Enterprise
Windows Server 2012 R2 (with Update 2919355): Essentials, Standard, Datacenter
Windows 7 SP1 (with latest Windows Updates): Home Premium, Professional, Enterprise, Ultimate